src/chain_spec.rs

```diff
@@ -117,33 +117,17 @@ impl Alternative {
     }
 }
 
+// Dev mode genesis setup
 fn testnet_genesis(initial_authorities: Vec<(AuraId, GrandpaId)>,
     root_key: AccountId,
     endowed_accounts: Vec<AccountId>,
-    _enable_println: bool) -> GenesisConfig {
-    let initial_utxo = utxo::TransactionOutput {
+    _enable_println: bool) -> GenesisConfig 
+{
+    let genesis_utxo = utxo::TransactionOutput {
```

Comment:
Might be nice to use the `endowed_accounts` parameter here. Provide an initial UTXO for each endowed account.

Reply:
Utxo doesn't have a notion of accounts. i.e. you can spend utxo even with unsigned extrinsics since the sigScript is in the input-utxo itself. Or do you mean something else by this?

Reply:
My point was that it would be nice to pass in a vec of identities who would receive UTXOs at genesis rather than hard-coding a single one for Alice. This would look similar to how you pass in a vec of accounts called `endowed_accounts`. At first I was thinking you could use the same `endowed_accounts` variable even though "accounts" is the wrong jargon. Looking closer now using that same variable won't work because UTXOs require sr25519 crypto while accounts can also be ed25519.

Concretely, I'm recommending something like this:
```rust
fn testnet_genesis(initial_authorities: Vec<(AuraId, GrandpaId)>,
    root_key: AccountId,
    endowed_accounts: Vec<AccountId>,
    utxo_endowed_identities: Vec<sr25519::Public>, // <-- Add this parameter
    _enable_println: bool) -> GenesisConfig
{
    GenesisConfig {
      // --snip--
      utxo: Some(utxo::GenesisConfig {
        genesis_utxo: utxo_endowed_identities
          .iter()
          .map(|x|
            utxo::TransactionOutput{
              value: utxo::Value::max_value(),
              pubkey: H256::from_slice(x.as_slice()),
          })
          .collect()
      }),
    }
}
```

Reply:
Great idea! I'll do that, giving everyone value 100 so there's no overflow during the demo.

Reply:
this is done